packages/utilities/src/utils/importDelegatedModule.ts

```diff
@@ -40,8 +40,7 @@ export const importDelegatedModule = async (
                             //@ts-ignore
                             `${keyOrRuntimeRemoteItem.global}->${arg}`
                           );
-                        // eslint-disable-next-line prefer-rest-params
```

Comment:
The one time eslint catches an actual bug

Reply:
LOL